src/app/containers/ATIAnalytics/canonical/index.jsx

```diff
@@ -21,7 +21,7 @@ const CanonicalATIAnalytics = ({ pageviewParams }) => {
         // lazy and didn't want to write a fuzzy matcher for the unit AND e2e
         // tests (you can't predict the class names chosen by styled-components)
         style={{ position: 'absolute' }}
-        src={atiPageViewUrl}
+        src={atiPageViewUrl.replace('x8=[simorgh]', 'x8=[simorgh-nojs]')}
```

Comment:
Just realised I think we need to add some logic around this in case the url doesn't have `'x8=[simorgh]'` because if it doesn't then `'x8=[simorgh-nojs]'` won't be switched in. I'd check for the presence of `'x8=[simorgh]'` in the string before replacing. If it doesn't have it in the string then add it on

Reply:
@jroebu14  Thats a good suggestion, although the logic is such that an `x8=[simorgh]` value is always present in the url. However I've added this check just as a failsafe.